UIforETW/UIforETWDlg.cpp

```diff
@@ -433,27 +1196,47 @@ BOOL CUIforETWDlg::OnInitDialog()
 	return TRUE; // return TRUE unless you set the focus to a control
 }
 
-std::wstring CUIforETWDlg::GetDirectory(const wchar_t* env, const std::wstring& default)
+void CUIforETWDlg::initializeToolTip( )

```

Comment:
Having this in a separate function is good. But what purpose is served by having the text declared as variables? It seems to open up new potential for errors, it pollutes the (anonymous) global namespace, and it puts a huge separation between the definition of the strings and their usage. I'm not clear on the benefit.